README.md

```diff
@@ -1,3 +1,135 @@
 # scats
-
 Selenium Cucumber automated testing system
+
```

Comment:
so ok, with the current setup, other projects can include scats-core jar into lib to use normally.